tools/cloud-build/daily-tests/builds/ctk-dockerfile.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+tags: [dockerfile, m.pre-existing-vpc, m.vm-instance, vm]
+
+timeout: 3600s  # 1hr
+steps:
+## uses a pre-built Google Cloud image containing Docker CLI tool.
+- id: build-docker-image
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcluster', 'tools/cloud-build/images/cluster-toolkit-dockerfile/']
+
+- id: create-deployment-folder
+  name: 'gcluster'
+  args:
+  - 'create'
+  - 'tools/cloud-build/daily-tests/blueprints/e2e.yaml'
+  - '--vars'
+  - 'project_id=$PROJECT_ID'
+  - '--vars'
+  - 'deployment_name=dockerfile-test'
+  - '--vars'
+  - 'region=us-central1'
+  - '--vars'
+  - 'zone=us-central1-a'
+  - '--out'
+  - '/workspace'
```

Comment:
nit: I think this might look cleaner and easier to read:

```yaml
args:
- 'create'
- 'tools/cloud-build/daily-tests/blueprints/e2e.yaml'
- '--out'
- '/workspace'
- '--vars'
- 'project_id=$PROJECT_ID,deployment_name=dockerfile-test,region=us-central1,zone=us-central1-a'
```